documentation/onboarding.md

```diff
@@ -16,7 +16,67 @@ Please read the [ARIA WG Process Document](process.md)
 * add **Channels**, "#aria"
 
 ### IRC command cheat sheet
-* [ARIA Issue 1404](https://github.com/w3c/aria/issues/1401#issuecomment-786384070)
+When you first enter the channel, type:
+`present+`
+
+Commands anyone can use:
+```
+q+  (to add yourself to the speaker queue)
+q+ to say [what you want to say]  (to add yourself to the speaker queue with reminder message)
+ack me  (take me off the q; or can use q-)
+ack ja  (take james off the q; or can use q- ja)
+q?  (to find out if anyone is on the q)
+
```

Comment:
should we add info about advanced queueing.... 
qq+ (add to front of queue - normally used to react to the current speaker)
q- later (move to end of the queue)
q+ later (add to queue a 2nd time often used if you want to react to something but also have another point to make)
If not we could just point to https://www.w3.org/2001/12/zakim-irc-bot.html#speakerqueue